### PR TITLE
add link to newest post

### PR DESCRIPTION
**Pull request by lbischof**

### CODE REVIEW

File: app/index/template.hbs [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div class="index-content">
   <section class="latest-contribution">
     {{#each (take 1 (sort-by 'insertedAt:desc' model.contributions))
-      as |latestContribution|}}
+      as |latestContribution|}}<a href="/day/{{latestContribution.date}}">
```
Comment (topaxi) [inline app/index/template.hbs]: In an ember app, you shouldn't use plain old html5 links, these result in a hardreload of the entire page.

Use
```handlebars
{{#link-to 'day' latestContribution.date}}<!-- link content -->{{/link-to}}
```
instead :)

Comment (lbischof) [inline app/index/template.hbs]: fixed

**Comment by topaxi** [comment]
Looks good now :)

Maybe check in with @Joshuasony as this change changes the color of the text to the link-color (before the change it was white).

**Comment by lbischof** [comment]
Thanks, it's white now :+1:

**Comment by topaxi** [comment]
Nice thanks!

**Pull request merged**